stl/inc/memory

```diff
@@ -14,6 +14,10 @@
 #include <typeinfo>
 #include <xmemory>
 
+#if _HAS_CXX20
+#include <atomic>

```

Comment:
Note to others: normally this would have to be guarded for `/clr:pure`, but `/clr:pure` implies not C++20 so that's already covered.